mkdocs.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
This isn't needed, please remove.

Reply:
It is needed by pre-commit added in: https://github.com/TrenchBoot/TrenchBoot.github.io/pull/37
I can remove it but it'll be added by that PR anyway.
yamlint can be configured to ignore this but it doesn't hurt or create hardly any additional work and I'd rather keep configuration consistent.

Reply:
Removed all pre-commit/indent fixes/commits as https://github.com/TrenchBoot/TrenchBoot.github.io/pull/37 would fix it anyway.